hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/distribution/DistributionTraitDef.java

```diff
@@ -115,37 +121,11 @@ public RelNode convert(
             case ANY:
                 return rel;
 
-            case ROOT:
```

Comment:
Why it was removed?

Reply:
it is not used anymore.

Reply:
I guess none of the options is used but yet they are there?

Reply:
Yes, they are.